common/autoware_component_interface_tools/package.xml

```diff
@@ -17,8 +18,17 @@
   <depend>tier4_system_msgs</depend>
   <depend>yaml_cpp_vendor</depend>
 
+  <test_depend>ament_cmake_ros</test_depend>
```

Comment:
```suggestion
  <test_depend>ament_cmake_ros</test_depend>
```

It seems that this is not required.
- Explained here: https://github.com/autowarefoundation/autoware.universe/pull/5815#discussion_r1423379486